### ISSUE TITLE
Funciones parte 3

### ISSUE THREAD

**Issue by mfgonzalez7** [IIC2115/Syllabus#123]
Hola, 
Dado que hubo una modificación en el enunciado, producto de la demora de las requests. Para las funciones de consultas a realizar para la parte 3 ¿qué se debe considerar como el archivo tiempo_estimado.csv?.
Esto dado que ya no es un archivo completo para todos los paraderos, si no que contiene los paraderos correspondientes a la ruta del servicio indicado por el usuario, entonces por ejemplo la función buses_llegando_a_paradero, ¿el argumento será un paradero que sabemos que existe dentro de la base creada en la parte 1, o se debe crear un nuevo archivo correspondiente a ese paradero?

Gracias, espero se entienda la pregunta.

**Comment by FGarridoV** [comment]
Exacto, asume que esa pregunta será solo para los paraderos de los que tengan información de la parte anterior.

Saludos!
